common/djangoapps/student/views.py

```diff
@@ -813,7 +825,8 @@ def dashboard(request):
         'redirect_message': redirect_message,
         'course_enrollments': course_enrollments,
         'course_optouts': course_optouts,
-        'message': message,
+        'account_activation_message': account_activation_message,
```

Comment:
Would we rather have a single `account_activation_message` string, with a side-channel `display_on_sidebar` variable, with logic in the template to determine where to put it, or two separate strings, either of which can be either blank or have an HTML value, which the template would simply render in different places if present? This seems a bit brittle to me, and necessitates extra logic in the template.